radix-engine/src/blueprints/consensus_manager/validator.rs

```diff
@@ -1214,7 +1231,7 @@ impl ValidatorBlueprint {
             .and_modify(|pending_amount| {
                 *pending_amount = pending_amount
                     .safe_add(requested_stake_unit_amount)
-                    .unwrap()
+                    .unwrap_or(Decimal::MAX)
```

Comment:
Not sure if I agree with this silently going to `Decimal::MAX` when we overflow instead of erroring out with something like "you have too much pending, claim some of them and then try again"

Reply:
Right, this should always fail later when we actually try to move the amount from the Locked stake vault to  Unlocking stake vault. I think it's more of a question of whether to early exit or not in the function due to an overflow.